### DISCUSSION TITLE
skywalking 8.7.0, page 'tracing' bug

### DISCUSSION THREAD

**Discussion by little-hang** [apache/skywalking#8627] [General]
When enter in page 'tracing'， and then choose the follows conditions， bug happens
![image](https://user-images.githubusercontent.com/24517540/157013288-9fe658e3-ac58-4a5d-a143-af097be141f8.png)

ps: the ui version---8.7.0

**Comment by wu-sheng** [comment]
Your version is old, the latest is 8.9.1. Also, new UI is being added in 9.0.
The community is not working on v8 anymore.
